README.md

```diff
@@ -71,6 +71,10 @@ Parameters for the script are controlled by the following environment variables:
 - `NVIDIAGPU_DEPLOY_FROM_BUNDLE`: boolean flag to deploy GPU operator from bundle image with operator-sdk - Default value is false - _required when deploying from bundle_
 - `NVIDIAGPU_SUBSCRIPTION_UPGRADE_TO_CHANNEL`: specific subscription channel to upgrade to from previous version.  _required when running operator-upgrade testcase_
 - `NVIDIAGPU_CLEANUP`: boolean flag to cleanup up resources created by testcase after testcase execution - Default value is true - _required only when cleanup is not needed_
+- `NVIDIAGPU_GPU_CREATE_CUSTOM_CATALOGSOURCE`: boolean flag to deploy a custom certified-operators catalogsource for GPU package - Default value is false - _optional_
+- `NVIDIAGPU_NFD_CREATE_CUSTOM_CATALOGSOURCE`: boolean flag to deploy a custom redhat-operators catalogsource for NFD package - Default value is false - _optional_
+- `NVIDIAGPU_GPU_CATALOGSOURCE_INDEX_IMAGE`: custom certified-operators catalogsource index image for GPU package - _required when deploying custom GPU catalogsource_
+- `NVIDIAGPU_NFD_CATALOGSOURCE_INDEX_IMAGE`:  custom redhat-operators catalogsource index image for NFD package - _required when deploying custom NFD catalogsource_
```

Comment:
I think these can be combined into just two variables. I.e. if a custom catalog source is specified, it will be used. IMO we don't need a flag here, as it complicates the code/configuration.

Reply:
Or do you want to optionally enable a custom catalog if an operator isn't available in the default catalog?

Reply:
The logic in the code is to create the custom (fallback) catalog if an operator is not available in its default catalog, and if the flags to create the catalog are set with the custom index image.
I'll use just two env variables instead and rename them to:
 `NVIDIAGPU_GPU_FALLBACK_CATALOGSOURCE_INDEX_IMAGE`
 `NVIDIAGPU_NFD_FALLBACK_CATALOGSOURCE_INDEX_IMAGE`
If they are set, then we will enable creation of custom catalogsources if the oeprators are not in the respective default catalogs,